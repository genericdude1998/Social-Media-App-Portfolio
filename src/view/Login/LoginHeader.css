.baseLoginHeader{
    width: 100vw;
    text-align: center;
    min-width: 600px;
}

img{
    height: 250px;
    width: 250px;
}