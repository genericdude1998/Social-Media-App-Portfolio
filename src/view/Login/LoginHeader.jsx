import React from 'react';
import styles from './LoginHeader.css'

const LoginHeader = () => {
    return (
        <div className={styles.baseLoginHeader}>
            Welcome to my Social Media App portfolio:
        </div>
    );
}

export default LoginHeader;
