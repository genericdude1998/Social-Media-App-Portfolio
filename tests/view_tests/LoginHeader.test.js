import React from 'react';
import { shallow } from 'enzyme';
import LoginHeader from '../../src/view/Login/LoginHeader';

let loginHeaderWrapper = shallow(
    <LoginHeader/>
);
it('should match the snapshot', () => {
    expect(loginHeaderWrapper).toMatchSnapshot();
});