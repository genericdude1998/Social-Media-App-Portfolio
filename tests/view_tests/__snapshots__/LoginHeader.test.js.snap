// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
<div
  className="baseLoginHeader"
>
  Welcome to my Social Media App portfolio:
</div>
`;
